connector/ldap/testdata/core.schema

```diff
@@ -1,610 +0,0 @@
-# OpenLDAP Core schema
```

Comment:
Weren't these files used at all?

Reply:
The earlier tests started slapd with a clean config. These schemas are already loaded into the docker version.

Reply:
👍